core/src/main/java/org/fao/geonet/kernel/search/SearchManager.java

```diff
@@ -1588,7 +1588,8 @@ private IndexInformation newDocument(String language, Element xml, Collection<Fi
         Classifier classifier = dimension.getClassifier();
 
         for (CategoryPath categoryPath: classifier.classify(value)) {
-            if (!dimension.isLocalized() || dimension.getLocales().contains(locale)) {
+            result.add(new FacetField(dimension.getName(), categoryPath.components));
```

Comment:
In fact it breaks indexing because a field and a facet have the same name. I will fix that.